elasticsearch.yml

```diff
@@ -0,0 +1,3 @@
+network.host: 0.0.0.0
```

Comment:
Is this just for the docker container in local?

Reply:
yes its